ftw/oidcauth/browser/oidc_tools.py

```diff
@@ -265,7 +265,7 @@ def update_login_times_and_other_member_properties(self, member):
     def setup_session(self):
         uf = api.portal.get_tool('acl_users')
         uf.updateCredentials(
-            self.request, self.request.response, self.userid, '')
+            self.request, self.request.response, safe_utf8(self.userid), '')
```

Comment:
Could this be tested with a userid containing an umlaut?

Reply:
I added a test that even helped me to improve/fix the implementation.